charts/datahub/subcharts/datahub-gms/values.yaml

```diff
@@ -151,6 +151,7 @@ global:
         secretKey: "mysql-password"
 
   datahub:
+    version: head
```

Comment:
@szalai1 Was the intent of the global version change to pin everything to a release version as well? I see other images set to v0.9.3 so we want those to line up.

Reply:
`global.datahub.version` in top level `values.yaml` will overwrite this, setting it to 0.9.3.